nucypher_ops/playbooks/recover_ursula_ops_data.yml

```diff
@@ -67,9 +67,8 @@
           " InstanceId:             {{instance_id}}\n
            publicaddress:          {{ursula_config.rest_host}}\n
            host_nickname:          {{ursula_hostname.stdout}}\n
-           eth_provider:           {{ursula_config.eth_provider_uri}}\n
-           pre_payment_provider:       {{ursula_config.pre_payment_provider}}\n
-           pre_payment_network:        {{ursula_config.pre_payment_network}}\n
+           eth_endpoint:           {{ursula_config.eth_endpoint}}\n
+           polygon_endpoint:       {{ursula_config.polygon_endpoint}}\n
```

Comment:
what is the purpose of line #78?

```
           _domain:                {{ursula_config.domain}}\n
```

I don't fully understand what this `ursula_config.domain` stands for, but it seems like it hasn't any relation with TACo domains (tapir, lynx...), so it can be confussing.

Also, it seems like this is not very used?

Maybe we could remove that line to avoid confussion or use it as the actual TACo domain.

WDYT?

Reply:
@manumonti apologies, I didn't see this comment before merging.

> I don't fully understand what this ursula_config.domain stands for, but it seems like it hasn't any relation with TACo domains (tapir, lynx...), so it can be confusing.

It's exactly that i.e. the TACo domain. It's the `domain` value stored in the `ursula.json` file on the remote machine. It's value can be `tapir`, `lynx`, `mainnet`.

This recovery playbook basically goes to the remote machine and gathers all of the information about the Ursula to regenerate the `nucypher-ops` configuration files. It's the remedy from when I did `rm -rf` on the config folder. Typically nucypher-ops has config files that it uses to connect and configure remote nodes. This is the opposite - there is no config file, but we connect to the machine to get the information to regenerate the configuration file.

Reply:
got it! Thanks